timeless/restaurants/models.py

```diff
@@ -81,6 +81,28 @@ def __init__(self, **kwargs):
     def __repr__(self):
         return "<Location %r>" % self.name
 
+    @classmethod
+    def merge_with_poster(cls, location, poster_location: dict):
+        """
+        Method should return Location object with merged data from table entity
+        and poster location dict
+        @todo #343:30min Implement two class methods merge_with_poster and
+         create_by_poster. merge_with_poster will merge entry entity with
+         poster entity, we should make right fields mapping, as result
+         returns Location instance.
+         The same should be made with method create_by_poster, returns Location
+         instance with data from poster_customer
+        """
+        return cls()
+
+    @classmethod
+    def create_by_poster(cls, poster_location: dict):
+        """
+        Method should return Location object with given data from
+        poster_location dict
+        """
+        return cls()
```

Comment:
@vladarefiev I think for now, these methods should throw UnimplementedException, instead of returning something.

Reply:
@amihaiemil ok